namerd/storage/consul/src/main/scala/io/buoyant/namerd/storage/consul/ConsulDtabStore.scala

```diff
@@ -19,6 +21,8 @@ class ConsulDtabStore(
 
   private[this] val log = Logger.get("consul")
 
+  def namespaceIsValid(ns: Ns): Boolean = Pattern.matches("[\\w+\\-?/?]+", ns)
```

Comment:
Oh, also, TIOLI, but if you use a [raw string](https://docs.scala-lang.org/overviews/core/string-interpolation.html#the-raw-interpolator) you won't have to escape the `\` characters in your regex, which might make it a little more readable.